crates/revm/Cargo.toml

```diff
@@ -21,4 +21,12 @@ reth-consensus-common = { path = "../consensus/common" }
 revm.workspace = true
 
 # common
-tracing.workspace = true
\ No newline at end of file
+tracing.workspace = true
+
+[dev-dependencies]
+reth-rlp.workspace = true
+
+# for creating keys, signing, mostly for eip4788 contract deployment / usage
+# tests
+reth-interfaces = { workspace = true, features = ["test-utils"] }
```

Comment:
in this test, which also sends a tx that calls the 4788 contract:
https://github.com/paradigmxyz/reth/blob/fed5b0e224cee1f72c7fbda4d53bb78026fde4c9/crates/revm/src/processor.rs#L627-L628

We can remove this part of the test to avoid the dep though, this part of the test is not testing anything critical, just that a regular tx can call the contract and succeed

Reply:
removed now